src/content/self-hosted/helm-configuration.mdx

```diff
@@ -200,6 +217,10 @@ The build service. It's used in combination with `okteto build` to build contain
 - `hpa.min`: Minimum number of buildkit pods to keep running.
 - `hpa.max`: Maximum number of buildkit pods to scale to.
 - `hpa.cpu`: The amount of CPU utilization that will cause the HPA to scale the buildkit pods.
+- `image`: Specifies the image settings for your buildkit service.
+  - `registry`: The registry where the buildkit image is hosted. Defaults to `` (You can change all the registries by setting `globals.registry` to your registry).
+  - `repository`: The repository path for the buildkit image. Defaults to `okteto/buildkit` if not specified.
+  - `tag`: The image tag to deploy. Defaults to `0.17.2-okteto1` if not specified.
```

Comment:
I don't like to set the default value here, it would imply changing it almost every release

Reply:
My understanding is that, as part of this work, we were tagging buildkit and registry with the chart release, didn't we do that?